pkg/util/util.go

```diff
@@ -156,6 +158,30 @@ func TruncateString(str string, maxLen int) string {
 	return str
 }
 
+// GetAbsPath returns absolute path from passed file path resolving even ~ to user home dir and any other such symbols that are only
+// shell expanded can also be handled here
+func GetAbsPath(path string) (string, error) {
```

Comment:
Could we have some unit tests for this function?

Reply:
+1

Reply:
Please add unit tests.

Reply:
Sure @geoand and @metacosm 
I'll update the PR with tests and comments incorporated

Reply:
Done